File: Assignment_43/program3.c
// Write a recursive program which accept string from user and count number of small characters.
// Input : HElloWOrlD 
// Output : 5 

///////////////////////////////////////////////////////////////////////////////
// File Name   :  program3.c
// Description :  Recursive program which accept string from user and count number of small charcters.
// Autor       :  Ankita Anil Patil
// Date        :  23/07/2025
///////////////////////////////////////////////////////////////////////////////

   
#include<stdio.h>
int Small(char *str) 
{ 
    static int iCount = 0;

    if(*str != '\0')
    {
        if((*str >= 'a') && (*str <= 'z'))
        {
            iCount++;
        }

        Small(++str);
    }

    return iCount;

} 
int main() 
{ 
    int iRet = 0; 
    char arr[20]; 

    printf("Enter string : "); 
    scanf("%[^'\n]s",arr); 

    iRet = Small(arr); 

    printf("%d",iRet); 

    return 0; 
}

File: Assignment_54/program3.java
// Write java program which accept array of characters from user and count number of capital characters. 
/*
    Input :  b N j B R b A d G G 
    Output : 6
 */

////////////////////////////////////////////////////////////////////////////////
// File Name   :  program3.java
// Description :  Accept array of characters from user and count number of capital characterss      
// Autor       :  Ankita Anil Patil
// Date        :  13/09/2025
///////////////////////////////////////////////////////////////////////////////


import java.util.*;

class MyArray 
{ 
        public int ArrayCapital(char arr[]) 
        { 
            int iCount = 0;
            for(char ch : arr)
            {
                if(ch >= 'A' && ch <= 'Z')
                {
                    iCount++;
                }
            }

            return iCount;
        }
} 
public class program3
    
{   
    public static void main(String[] args) {
        

    Scanner sobj = new Scanner(System.in);

    System.out.println("Enter size of array");
    int iSize = sobj.nextInt();


    char[] Arr = new char[iSize];
    System.out.println("Enter charcater : ");

    int i = 0;
    
    for(i = 0; i < Arr.length; i++)
    { 
       Arr[i] = sobj.next().charAt(0);
    }

    MyArray mobj = new MyArray();
    int iRet = mobj.ArrayCapital(Arr);

    System.out.println("Number of capital letter in array are :"+iRet);
    
}
}
